### ISSUE TITLE
Async generators

### ISSUE THREAD

**Issue by mnieper** [google/traceur-compiler#1673]
Has https://github.com/jhusain/asyncgenerator progressed far enough so that one may implement at least the fundamental bits of it in Traceur?

**Comment by arv** [comment]
Yeah. It seems like it would be really useful to have a prototype of this in Traceur (under a flag of course).
